src/rules/converters/eslint-plugin-react/tests/jsx-equals-spacing.test.ts

```diff
@@ -0,0 +1,46 @@
+import { convertJsxEqualsSpacing } from "../jsx-equals-spacing";
+
+describe(convertJsxEqualsSpacing, () => {
+    test("conversion without arguments", () => {
+        const result = convertJsxEqualsSpacing({
+            ruleArguments: [],
+        });
+
+        expect(result).toEqual({
+            rules: [
+                {
+                    ruleName: "react/jsx-equals-spacing",
+                },
+            ],
+            plugins: ["eslint-plugin-react"],
+        });
+    });
+
+    test("conversion with arguments", () => {
+        const alwaysResult = convertJsxEqualsSpacing({
+            ruleArguments: ["always"],
+        });
+        const neverResult = convertJsxEqualsSpacing({
+            ruleArguments: ["never"],
+        });
+
+        expect(alwaysResult).toEqual({
+            rules: [
+                {
+                    ruleArguments: ["always"],
+                    ruleName: "react/jsx-equals-spacing",
+                },
+            ],
+            plugins: ["eslint-plugin-react"],
+        });
```

Comment:
🤔  looks like this is combining multiple test cases into a single unit test. That's a little sub-optimal from a developer experience perspective because if the first fails, we don't get info from the second. Would you mind splitting these into two tests please?